.github/workflows/lint-pr.yml

```diff
@@ -0,0 +1,22 @@
+name: "Lint PR"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
```

Comment:
why do we need "synchronize" here?

Reply:
I actually just copied it from the SDK config.